src/fides/api/alembic/migrations/versions/57479150efd6_add_property_id_to_historical_records.py

```diff
@@ -0,0 +1,60 @@
+"""add property id to historical records
+
+Revision ID: 57479150efd6
+Revises: fc2b2c06e595
+Create Date: 2024-05-14 18:26:57.646508
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "57479150efd6"
+down_revision = "fc2b2c06e595"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        "privacypreferencehistory", sa.Column("property_id", sa.String(), nullable=True)
+    )
+    with op.get_context().autocommit_block():
```

Comment:
Concurrent index creation cannot happen inside a transaction so this needs to auto-commit immediately.

Reply:
I missed discussion on this one - My gut opinion is that a migration should not auto-commit.   

Sometimes we are upgrading customers across multiple fides versions and multiple migrations, and if something goes wrong with the upgrade, it is fairly straightforward to bail early and roll them back, because the migrations themselves did not commit (doesn't happen until they're all done).

In my opinion, I'd have the standard image add the index per usual, not concurrently, but then create special images for large customers that omits the index creation.  Then we separately in a postgres shell create the index by hand.

This is just one thought, but I worry the auto-commit here is dangerous.